ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-security.j2

```diff
@@ -45,7 +45,11 @@
 {%             set dpd_time = profile.dpd_time | arista.avd.default("-") %}
 {%             set dpd_action = profile.dpd_action | arista.avd.default("-") %}
 {%             set mode = profile.mode | arista.avd.default("-") %}
-| {{ profile.name }} | {{ ike_policy }} | {{ sa_policy }} | {{ connection }} | {{ dpd_interval }} | {{ dpd_time }} | {{ dpd_action }} | {{ mode }} |
+{%             set flow_parallelization = "Disabled" %}
+{%             if profile.flow_parallelization_encapsulation_udp is arista.avd.defined(true) %}
+{%                 set flow_parallelization = "Enabled" %}
+{%             endif %}
```

Comment:
Normally we use `-` for unset like you see in the lines above.
I am OK using `True` as the positive value, so this can be simplified to:
```suggestion
{%             set flow_parallelization = profile.flow_parallelization_encapsulation_udp | arista.avd.default('-') %}
```

Reply:
Done.